api/envoy/config/cluster/v3/outlier_detection.proto

```diff
@@ -173,4 +173,8 @@ message OutlierDetection {
   // Set of host's passive monitors.
   // [#not-implemented-hide:]
   repeated core.v3.TypedExtensionConfig monitors = 24;
+
+  // If enabled, at least one host is ejected regardless of the value of :ref:`base_ejection_time<envoy_v3_api_field_config.cluster.v3.OutlierDetection.max_ejection_percent>`.
+  // Defaults to false.
+  google.protobuf.BoolValue may_always_eject_one = 25;
```

Comment:
Can you rename it to `always_eject_one_host`?

Reply:
yup, this sounds better!
`always_allow_one_ejection` is another candidate

Reply:
updated.